Cargo.toml

```diff
@@ -26,14 +26,17 @@ harness = false
 rust_decimal = "1.32"
 rust_decimal_macros = "1.32"
 lexers = "0.1.4"
-nom = "7.1.3"
 unicode-segmentation = "1.10.1"
 chrono = "0.4.31"
 url = "2.4.1"
+# This is for the forest tree implementation
 #generational-arena = "0.2"
 regex = "1.10.2"
 
 [dev-dependencies]
 criterion = "0.5.1"
 encoding_rs = "0.8.33"
-encoding_rs_io = "0.1.7"
\ No newline at end of file
+encoding_rs_io = "0.1.7"
+#ixml = { package = "earleybird", version = "0.0.1" }
+earleybird = {path = "../earleybird"}
```

Comment:
Could we set this to 

earleybird = {git = "https://github.com/mdubinko/earleybird.git"}

instead? Will automatically resolve the dependency then.

Reply:
OK, I'll try that

Reply:
Done. This has been set in the new-features-2023-11-28 branch.